src/DotNet/Pdb/Managed/DbiModule.cs

```diff
@@ -157,7 +157,7 @@ sealed class DbiModule {
 			if (funcs[found].Lines is null) {
 				while (found > 0) {
 					var prevFunc = funcs[found - 1];
-					if (prevFunc is not null || prevFunc.Address != address)
+					if (prevFunc is not null && prevFunc.Address != address)
```

Comment:
I'm pretty sure this was an unintentional mixup of the `||` and `&&` operators which would result in right-hand side being executed only if `prevFunc` was `null` causing an NRE. If I'm wrong feel free to correct me.